src/CLI/actioner/sonic-cli-tacacs.py

```diff
@@ -0,0 +1,146 @@
+       return api.delete(path)
+    elif func == 'delete_openconfig_system_augments_system_aaa_server_groups_server_group_config_timeout':
+       path = cc.Path('/restconf/data/openconfig-system:system/aaa/server-groups/server-group=TACACS/config/openconfig-system-augments:timeout')
+       return api.delete(path)
+    elif func == 'delete_openconfig_system_augments_system_aaa_server_groups_server_group_config_auth_type':
+       path = cc.Path('/restconf/data/openconfig-system:system/aaa/server-groups/server-group=TACACS/config/openconfig-system-augments:auth-type')
+       return api.delete(path)
+    elif func == 'delete_openconfig_system_augments_system_aaa_server_groups_server_group_config_secret-key':
+       path = cc.Path('/restconf/data/openconfig-system:system/aaa/server-groups/server-group=TACACS/config/openconfig-system-augments:secret-key')
+       return api.delete(path)
+    else:
+       body = {}
+
+    return api.cli_not_implemented(func)
+
+def get_sonic_tacacs_server(args):
+    api_response = {}
+    api = cc.ApiClient()
+
+    path = cc.Path('/restconf/data/openconfig-system:system/aaa/server-groups/server-group=TACACS/servers/server={address}/tacacs/config', address=args[0])
+    response = api.get(path)
+    if response.ok():
+        if response.content:
+            api_response['port'] = response.content['openconfig-system:config']['port']
+            api_response['key'] = response.content['openconfig-system:config']['secret-key']
+
+    path = cc.Path('/restconf/data/openconfig-system:system/aaa/server-groups/server-group=TACACS/servers/server={address}/config', address=args[0])
+    response = api.get(path)
+    if response.ok():
+        if response.content:
+            api_response['authtype'] = response.content['openconfig-system:config']['openconfig-system-augments:auth-type']
+            api_response['priority'] = response.content['openconfig-system:config']['openconfig-system-augments:priority']
+            api_response['timeout'] = response.content['openconfig-system:config']['timeout']
+
+    api_response['address'] = args[0]
+    show_cli_output("show_tacacs_server.j2", api_response)
+
+def get_sonic_tacacs_global():
+    api_response = {}
+    api = cc.ApiClient()
+
+    path = cc.Path('/restconf/data/openconfig-system:system/aaa/server-groups/server-group=TACACS/config')
+    response = api.get(path)
+    if response.ok():
+        if response.content:
+            api_response = response.content
+
+    show_cli_output("show_tacacs_global.j2", api_response)
+
+def run(func, args):
+    response = invoke_api(func, args)
+    if response.ok():
+        if response.content is not None:
+            # Get Command Output
+            api_response = response.content
+
+            if api_response is None:
+                print("Failed")
+            else:
+                return
+    else:
+        print response.error_message()
```

Comment:
In python 3, print has become a function.

Reply:
Done